includes/editor-templates/library-layout.php

```diff
@@ -12,7 +12,9 @@
 				<# if ( 'skip' === closeType ) { #>
 				<span><?php echo esc_html__( 'Skip', 'elementor' ); ?></span>
 				<# } #>
-				<i class="eicon-close" aria-hidden="true"></i>
+				<i class="eicon-close"
+				   aria-hidden="true"
+				   title="{{{ $e.components.get( 'document/elements' ).utils.getTitleForLibraryClose() }}}"></i>
 				<span class="elementor-screen-only"><?php echo esc_html__( 'Close', 'elementor' ); ?></span>
```

Comment:
`<span class="elementor-screen-only"><?php echo esc_html__( 'Close', 'elementor' ); ?></span>`

Should this not also use the config?